apps/daimo-mobile/src/storage/storedTypes.ts

```diff
@@ -0,0 +1,245 @@
+import {
+  AddrLabel,
+  BigIntStr,
+  DaimoNoteState,
+  DaimoRequestState,
+  DollarStr,
+  OpStatus,
+} from "@daimo/common";
+import { Address, Hex } from "viem";
+
+//
+// Stored types used inside of StoredModels.
+//
+// Do not edit these except to add new OPTIONAL fields or enums.
+// Anything else requires a new version and a migration.
+// This file contains current types. For old versions, see storedTypeMigrations.
+//
+
+export type StoredV16Clog = StoredV16TransferClog | StoredV15PaymentLinkClog;
```

Comment:
note in future: change to`= SimpleTransferClog | PaymentLinkClog | SwapClog` -- which would remove `preSwapTransfer` from `TransferClog` and that whole clog will instead be a `SwapClog`